c10/core/impl/LocalDispatchKeySet.h

```diff
@@ -37,15 +37,15 @@ struct C10_API PODLocalDispatchKeySet {
   // See Note [TLS Initialization]
   DispatchKeySet included() const {
     return DispatchKeySet(DispatchKeySet::RAW, included_) ^
-        c10::default_included_set;
+        c10::default_included_set.load();
```

Comment:
I believe `memory_order_relaxed` is acceptable here. The model for when it is acceptable to change the default included set is at the very start of the program, before any threads hitting the dispatcher have begun. In this case, there is an implicit synchronization between the setter and all subsequent uses, and therefore no race is possible and relaxed is OK.